.gitignore

```diff
@@ -12,6 +12,8 @@
 /_install_pkgprotodir/
 Makefile
 Makefile.in
+CMakeCache.txt
+CMakeFiles/
```

Comment:
Did you (locally) have any CMake recipes to build NUT? In the future, such contribution might be welcome too, although currently the autoconf-based scriptware does the job and is quite large to make a 1:1 equivalent too complex to achieve.

Or was NUT source built for you as part of some larger cmake-driven project which left noise in your workspace?

Reply:
I built it locally to experiment with Arduino board. Sorry, I'm still new to CICD. Please kindly advise what I need to change, I will do necessary fixes and resubmit.

Reply:
With this bit, I was rather curious why you had to git-ignore cmake related files at all, since NUT does not dabble in that build system currently. Might as well stay ignored for the scratch files, in case someone adds real cmake recipes later on.

Reply:
@jimklimov I think I understood your question finally, sorry it took time :) I believe I tried to ccmake when attempting to build nut from source and this created garbage files, which I then tried to exclude through gitignore. I will revert this change in the next commit.